config/rootfs/debos/scripts/strip.sh

```diff
@@ -44,6 +44,9 @@ apt-get autoremove --yes || true
 # Dropping logs
 rm -rf /var/log/*
 
+# Remove .deb files we installed
+rm -f /var/cache/apt/archives/*.deb
```

Comment:
```suggestion
apt-get clean
```

There is actually an apt invocation for this